ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -39,9 +39,12 @@ Rectangle {
                 chatsModel.sendImage(sendImageArea.image);
             }
 
-            if(txtData.text.trim().length > 0){
-                let msg = interpretMessage(txtData.text.trim()).trim()
-                chatsModel.sendMessage(msg, chatColumn.isReply ? SelectedMessage.messageId : "", Utils.isOnlyEmoji(txtData.text) ? Constants.emojiType : Constants.messageType);
+            var msg = chatsModel.plainText(Emoji.deparse(txtData.text).trim()).trim()
```

Comment:
Seems like we do the send parsing both for this function and for the ChatButton (ChatButtons.qml line 34)
It would be better if they both called a `sendMessage` function that then did the parsing etc. 
As far as I can see, the ChatButtons function seems to have less features too (no sound for example)